dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -0,0 +1,66 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
```

Comment:
There's nothing I would change here, but I did want to touch on these environment variable related assignments. These are not coming out of the airflow secret store, they are coming out of the airflow deployment itself. They come up from the `.env` file that is in the root of the checkout.

I like @johnclary's idea of having a helper function that we can use to abstract the retrieval of secrets down to a single call. That function would need access to these secrets, but that's totally fine. It would reduce the amount of boiler plate code that we see in every DAG, and I think that's never a bad thing. I'm 👍 on the idea.

Reply:
Agreed - I created https://github.com/cityofaustin/atd-data-tech/issues/12432 to capture this while we still have just a few DAGs to update with a util.